application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -330,4 +333,13 @@ public String getOpenaiApiKey() {
     public String getSourceCodeBaseUrl() {
         return sourceCodeBaseUrl;
     }
+
+    /**
+     * The configuration about jshell.
+     * 
+     * @return the jshell configuration
+     */
```

Comment:
worthless javadoc. please improve it

Reply:
I added it to be consistant, since all other getter have javadoc even if it is worthless, do I still remove it ? I don't know what I can really add to it.

Reply:
the others are not worthless, they add extra information - or they are bad as well.

u cant remove it, since its public. please put in some effort and describe what "jshell" is and does in the context of tj-bot. the config is the place where people will read about it the first time and they have to understand what this config is about. i.e. that its an external api that executes java code using JShell.